aws/keyspaces/keyspaces.go

```diff
@@ -0,0 +1,68 @@
+// Package keyspaces is for working with AWS Keyspaces (for Apache Cassandra).
+package keyspaces
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gocql/gocql"
+	"github.com/pkg/errors"
+)
+
+type Keyspaces struct {
+	Session *gocql.Session
+}
+
+var service_username = os.Getenv("KEYSPACE_USER")
+var service_password = os.Getenv("KEYSPACE_PW")
+
+// New returns a Keyspaces struct which wraps a Keyspaces session. certPath
+// is the path of the required Starfield digital certificate to connect
+// to Keyspaces using SSL/TLS
+// (found here: https://certs.secureserver.net/repository/sf-class2-root.crt)
+func New(certPath string) (Keyspaces, error) {
+
+	ksClient := Keyspaces{}
+
+	var region string
+	if region = os.Getenv("AWS_REGION"); region == "" {
+		return ksClient, errors.New("AWS_REGION is not set")
+	}
+	if service_username == "" {
```

Comment:
Could the `service_username` and `service_password` variables be treated just like the region above rather than being global values that aren't exported?